docker-storage-setup.1

```diff
@@ -36,9 +36,10 @@ STORAGE_DRIVER:
       overlay2 and "". Empty string tells docker-storage-setup to
       not do any storage setup.
 
-EXTRA_DOCKER_STORAGE_OPTIONS:
+EXTRA_STORAGE_OPTIONS:
       A set of extra options that should be passed to the Docker
```

Comment:
But these tools will set environment for different tools, so the admin/packager will know the difference.   OCID-storage-setup will have defaults that explains these flags.